src/ds/d400/d400-private.h

```diff
@@ -155,6 +161,7 @@ namespace librealsense
             {RS435_RGB_PID, "5.8.15.0" },
             {RS405U_PID, "5.8.15.0" },
             {RS435I_PID, "5.12.7.100" },
+            {RS436I_PID, "5.16.3.100" },
```

Comment:
Please add a TODO update the final FW version

Reply:
Done